### PR TITLE
Add tether gun

### PR DESCRIPTION
**Pull request by metalgearsloth**
Off-brand gravity gun.

https://github.com/space-wizards/space-station-14/assets/31366439/245fb4f9-d137-4728-910a-011b11afb324

Still needs a lot of polishing but it generally works atm.

Requires https://github.com/space-wizards/RobustToolbox/pull/4041

:cl: metalgearsloth & Kheprep
- add: Added a tether gun for science.

### CODE REVIEW

**Comment by deltanedas** [comment]
does this attach to portals

**Comment by metalgearsloth** [comment]
> does this attach to portals

No. Only dynamic or anchored bodies if admin (portals are static but not anchored).

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Launchers/launchers.yml [hunk 1]
```diff
@@ -151,6 +151,22 @@
       soundInsert:
         path: /Audio/Weapons/Guns/Gunshots/grenade_launcher.ogg
 
+- type: entity
+  name: tether gun
+  parent: BaseItem
+  id: WeaponTetherGun
+  description: Manipulates gravity around object to fling them at high velocities.
```
Comment (deltanedas) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Launchers/launchers.yml:158]: objects

File: Resources/Textures/Objects/Weapons/Guns/Launchers/tether_gun.rsi/meta.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{
+	"version": 1,
```
Comment (deltanedas) [inline Resources/Textures/Objects/Weapons/Guns/Launchers/tether_gun.rsi/meta.json:2]: this files indent looks hyper sus

**Review by deltanedas — CHANGES_REQUESTED** [review]

**Comment by metalgearsloth** [comment]
beaby for bebiew

**Comment by EmoGarbage404** [comment]
Add it to RND, coward

**Comment by OttoMaticode** [comment]
A fellow shipbreaker, I see

**Comment by crazybrain23** [comment]
> 

At the very least, add it to the recipes but with no unlock, make it a tech disk gamble only.

**Comment by EmoGarbage404** [comment]
> At the very least, add it to the recipes but with no unlock, make it a tech disk gamble only.

Nah just chuck it in T2 experimental

**Review by Emisse — APPROVED** [review]

**Pull request merged**